integration/mocha-tests/tests/AudioTest.js

```diff
@@ -0,0 +1,282 @@
+const { v4: uuidv4 } = require('uuid');
+const { Window } = require('../utils/Window');
+const WebDriverFactory = require('../utils/WebDriverFactory');
+const SdkBaseTest = require('../utils/SdkBaseTest');
+const AudioTestPage = require('../pages/AudioTestPage');
+const { Logger, LogLevel } = require('../utils/Logger');
+
+/*
+ * 1. Starts a meeting
+ * 2. Adds 2 participants to the meeting
+ * 3. Turns on the audio tone for both
+ * 4. One attendee plays random tone
+ * 5. Checks if the other participant is able to hear the tone
+ * 6. Same attendee mutes the audio
+ * 7. Checks if the other participant is not able to hear the audio
+ * */
+describe('AudioTest', async function () {
+  describe('run test', async function () {
+    let driverFactoryOne;
+    let driverFactoryTwo;
+    let baseTestConfigOne;
+    let baseTestConfigTwo;
+    let pageOne;
+    let pageTwo;
+    let test_window;
+    let monitor_window;
+    let test_attendee_id;
+    let monitor_attendee_id;
+    let meetingId;
+    let numberOfSessions;
+    let failureCount;
+
+    before(async function () {
+      this.logger = new Logger('AudioTest');
+      this.logger.log('Retrieving the base test config');
+      baseTestConfigOne = new SdkBaseTest();
+      this.logger.log('Configuring the webdriver');
+      driverFactoryOne = new WebDriverFactory(
+        baseTestConfigOne.testName,
+        baseTestConfigOne.host,
+        baseTestConfigOne.testType,
+        baseTestConfigOne.url
+      );
+      await driverFactoryOne.build();
+      this.logger.log('Using the webdriver, opening the browser window');
+      numberOfSessions = driverFactoryOne.numberOfSessions;
+      driverFactoryOne.driver.manage().window().maximize();
+      this.logger.log('Instantiating selenium helper class');
+      pageOne = new AudioTestPage(driverFactoryOne.driver, this.logger);
+      failureCount = 0;
+    });
+
+    afterEach(async function () {
+      if (this.currentTest.state === 'failed') failureCount += 1;
+    });
+
+    after(async function () {
+      this.logger.log('Closing the webdriver');
+      const passed = failureCount === 0;
+
+      await driverFactoryOne.quit(passed);
+
+      if (passed === true) {
+        this.logger.log('AudioTest passed!!!', LogLevel.SUCCESS);
+        process.exit(0);
+      } else {
+        this.logger.log('AudioTest failed!!!', LogLevel.ERROR);
+        process.exit(1);
+      }
+    });
+
+    describe('on single session', async function () {
+      before(async function () {
+        if (numberOfSessions !== 1) {
+          this.logger.log(
+            `Skipping single session test because number of sessions required is ${numberOfSessions}`,
+            LogLevel.WARN
+          );
+          this.skip();
+        }
+      });
+
+      afterEach(async function () {
+        this.logger.printLogs();
+      });
+
+      describe('setup', async function () {
+        it('should open the meeting demo in two tabs', async function () {
```

Comment:
Should not have to do this and just have one single code flow for single vs multiple sessions. Maybe we can create a helper fucntion to map the window and page etc.